pkgs/development/interpreters/ruby/skip-installing-bundled-gems-2.7.patch

```diff
@@ -0,0 +1,25 @@
+From 4db7f89cec2386e09d1cf5642bc4768eab090cc7 Mon Sep 17 00:00:00 2001
+From: Andrew Childs <andrew.childs@bibo.com.ph>
+Date: Thu, 10 Mar 2022 16:13:44 +0900
+Subject: [PATCH] Skip installing bundled gems
+
+---
+ tool/rbinstall.rb | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/tool/rbinstall.rb b/tool/rbinstall.rb
+index 060390626f..bce2466597 100755
+--- a/tool/rbinstall.rb
++++ b/tool/rbinstall.rb
+@@ -863,6 +863,8 @@ def install_default_gem(dir, srcdir)
+ end
+ 
+ install?(:ext, :comm, :gem, :'bundled-gems') do
```

Comment:
Could we set some flag to skip the installation of bundled gems instead of doing this patch? I am thinking in terms of long term maintenance, since each patch we add means that there is a possibility that in the future the patch will fail to apply and someone will have to go there and update the patches.